validation/pipeline/qase/reporter-v2/main.go

```diff
@@ -309,7 +315,13 @@ func createRunDescription(buildUrl string) string {
 		description.WriteString(versions)
 	}
 
-	return description.String()
+	s := description.String()
+
+	if len(s) <= descriptionLimit {
+		return s
```

Comment:
nit; since this if statement is right before the return, why not say `if len > descriptionLimit; s = s[:descriptionLimit]` to avoid nested returns